tests/e2e/performance/multi_snapshots.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import time
+import datetime
+import yaml
+import tempfile
+import logging
+import subprocess
+
+from ocs_ci.ocs import constants
+
+ERRMSG = 'Error in command'
+
+log = logging.getLogger(__name__)
+
+# Dictionary to hold all test parameters that need to passed as environment
+# variables.
+# After all variables are read an validate, some more variables can be added
+# to this dictionary.
+params = {'KUBECONFIG': None, 'SNAPNUM': None, 'LOGPATH': None,
+          'FILESIZE': None, 'NSPACE': None, 'INTERFACE': None,
+          'PODNAME': None, 'PVCNAME': None}
+
+# Dictionary to hold the names of pods which holding logs of creation time
+log_names = {'start': None, 'end': []}
+
+snap_yaml = None  # define this parameter as global
+format = '%H:%M:%S.%f'
+
+log_file_name = os.path.basename(__file__).replace('.py', '.log')
+
+
+def msg_logging(msg):
+    """
+    This function is logging the message to the log file, and also print it
+    for the caller script output
+
+    Args:
+        msg (str): The message to log as info and print on the console
+
+    """
+    print(msg)
+    log.info(msg)
+
+
+def run_command(cmd):
+    """
+    Running command on the OS and return the STDOUT & STDERR outputs
+    in case of argument is not string or list, return error message
+
+    Args:
+        cmd (str/list): the command to execute
+
+    Returns:
+        list : all STDOUT / STDERR output as list of lines
+
+    """
+    if isinstance(cmd, str):
+        command = cmd.split()
+    elif isinstance(cmd, list):
+        command = cmd
+    else:
+        return ERRMSG
+
+    msg_logging(f'Going to run {cmd}')
+    completed_process = subprocess.run(
+        command,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        stdin=subprocess.PIPE,
+        timeout=600
+    )
+    output = completed_process.stdout.decode()
+    err = completed_process.stderr.decode()
+    # exit code is not zero
+    if completed_process.returncode:
+        log.error(
+            f'Command finished with non zero {completed_process.returncode}'
+        )
+        output += f'{ERRMSG} {err}'
+
+    output = output.split('\n')  # convert output to list
+    output.pop()  # remove last empty element from the list
+    return output
+
+
+def run_oc_command(cmd, namespace):
+    """
+    Running an 'oc' command
+
+    Args:
+        cmd (str): the command to run
+        namespace (str): the namespace where to run the command
+
+    Returns:
+        list : the results of the command as list of lines
+
+    """
+    command = (
+        f'oc --kubeconfig {params["KUBECONFIG"]} '
+        f'-n {namespace} {cmd}'
+    )
+    return run_command(command)
+
+
+def run_cmd_on_pod(command, pod_name):
```

Comment:
same here. why not use `exec_oc_cmd` and manipulate the output to the format needed here?

Reply:
same as above.